.github/workflows/code-coverage.yml

```diff
@@ -0,0 +1,70 @@
+name: Code coverage
+on:
+  push:
+  pull_request:
+
+jobs:
+  codecoverage:
+    env:
+      PY_COLORS: "1"
+      source_directory: "./source"
+      collection_pre_install: >-
+        git+https://github.com/ansible-collections/ansible.utils.git
+        git+https://github.com/ansible-collections/ansible.netcommon.git
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-latest
+        ansible-version:
+          - stable-2.14
```

Comment:
Since this test is executed only to obtain the code coverage, it is run on the latest ansible version.

While adding unit test jobs in the workflow, those will be run on all supported ansible and python versions.

Reply:
That makes sense. Thanks for explaining.